frontend/src/common/mock/authCode/authCode.ts

```diff
@@ -0,0 +1,12 @@
+import { http, HttpResponse } from 'msw';
+
+import { API_ENDPOINTS } from '../../constants/apiEndpoints';
+
+const BASE_URL = process.env.BASE_URL;
+const AHTH_CODE_URL = `${BASE_URL}${API_ENDPOINTS.AUTH_CODE}`;
+
+const postAuthCode = http.post(AHTH_CODE_URL, async () => {
+  return HttpResponse.json({ message: '인증코드 발송 성공' }, { status: 201 });
```

Comment:
p4: await이 붙으면 에러가 발생할 때도 추적하기 편할 거 같아 👍

```suggestion
  return await HttpResponse.json({ message: '인증코드 발송 성공' }, { status: 201 });
```

---

참고자료
- https://jojoldu.tistory.com/699
- [msw 공식문서](https://mswjs.io/docs/http/intercepting-requests/body)

Reply:
덕분에 새로운 사실 배워가! 참고자료 굿굿👍 반영할게~~

Reply:
[refactor: Promise 타입 return하는 곳에 await 추가](https://github.com/woowacourse-teams/2025-Fit-toring/pull/274/commits/9fd7ebe904db1c0a4bc54716699fede13da986c6)